Client/Frontend/Shields/ShieldsViewController.swift

```diff
@@ -144,6 +144,7 @@ class ShieldsViewController: UIViewController, PopoverContentComponent {
     
     override func loadView() {
         view = View()
+        shieldsView.set(theme: Theme.of(tab))
```

Comment:
we can't use `applyTheme/styleChildren` here?

Reply:
We can, fixed.